src/main/kotlin/racingcar/Round.kt

```diff
@@ -0,0 +1,32 @@
+package racingcar
+
+class Round(carNames: List<String>) {
+    private var cars: MutableList<Car> = carNames.map { Car(it) }.toMutableList()
+
+
+    fun move() {
+        cars.forEach {
+            it.moveForward()
+        }
+        println()
+    }
+
+    fun printRound() {
+        cars.forEach {
+            val currentPosition = it.movingState()
+            println(currentPosition)
+        }
```

Comment:
```suggestion
        cars.forEach {
            println(it.movingState())
        }
```

변수로 받지 않고 바로 출력해주셔도 좋을 것 같습니다!

Reply:
가독성을 위해 유지하는 것도 나쁘지 않을 것 같네요... 아니면 it을 다른 이름으로 네이밍해준다든가...

Reply:
코드로 직접 예시로 보여주셔서 너무 감사드립니당 👍